jobbergate-cli/jobbergate_cli/subapps/job_scripts/app.py

```diff
@@ -244,21 +264,42 @@ def render(
         title="Created Job Script",
     )
 
-    # `submit` will be `None` --submit/--no-submit flag was not set
-    if submit is None:
-        # If not running in "fast" mode, ask the user what to do.
-        if not fast:
-            submit = typer.confirm("Would you like to submit this job immediately?")
+    # `submit` will be `None` when --submit/--no-submit flag was not set
+    if submit is None and (cluster_name or execution_directory):
+        # If the user specified --cluster-name or --execution-directory, they must also specify --submit
+        raise Abort(
+            "You must specify --cluster-name and --execution-directory only on submit mode.",
+            subject="Incorrect parameters",
+            support=True,
+        )
 
-        # Otherwise, assume that the job script should be submitted immediately
-        else:
-            submit = True
+    """
+    Invoke `question_helper` to handle the logic for parameters that are not specified by the user.
+
+    If the user specified --fast, the `question_helper` will not invoke the function and will
+    instead return the default answer. If the user did not specify --fast, the `question_helper`
+    will invoke the function and return the user's answer.
+    """
+
+    submit = question_helper(typer.confirm, "Would you like to submit this job immediately?", True, fast, submit)
+    download = question_helper(typer.confirm, "Would you like to download the job script files?", True, fast, download)
```

Comment:
I see the default value for `execution_directory` and `cluster_name` are handled in the create submission level, but it would be nice to ask the user for them here as well, just like download and submit

Reply:
Done!